OracleWebLogic/samples/12212-msiserver/README.md

```diff
@@ -94,35 +87,18 @@ So "simple_filename" helps identify the name of the file where the source is cop
 Using swarm service creation with this image
 --------------------------------------------
 The image called 12212-summercamps-msiserver can be used for Docker service creation
-to scale out to multiple replicas. docker service creation can only be done
-using an image from a docker registry. If you don't have publishing rights to a docker
+to scale out to multiple replicas. Docker service creation can only be done
+using an image from a Docker registry. If you don't have publishing rights to a Docker
 registry you can start one locally. More details can be found here
 https://docs.docker.com/registry/deploying/
 
-Here are the three steps you'll need to start, use and push to your local registry.
-If you do have push rights to an existing docker registry, you can skip to the third
-step
-
-1. Start the registry
-docker run -d -p 5000:5000 --restart=always --name registry registry:2
-
-2. Change docker preferences to add this local registry to your list of insecure registries
-{
-  "insecure-registries" : [
-    "localhost:5000"
-  ]
-}
-
-3. Push to the local registry
+Here is an example of how this image can be pushed to the local registry

```

Comment:
Please remove lines 95-97. We shouldn't be documenting how to use a local registry. That's handled by our own documentation as well as the upstream documentation and is out of scope here.